packages/components/src/internal/components/user/SiteUsersGridPanel.tsx

```diff
@@ -266,56 +302,66 @@ export class SiteUsersGridPanel extends React.PureComponent<Props, State> {
                         </MenuItem>
                     )}
                 </ManageDropdownButton>
-            </>
+            </div>
         );
     };
 
-    render() {
-        const { newUserRoleOptions, allowDelete } = this.props;
+    render(): ReactNode {
+        const { newUserRoleOptions, user, showDetailsPanel, actions } = this.props;
         const { selectedUserId, showDialog, usersView } = this.state;
+        const model = this.getUsersModel();
+        const modelLoading = !model || isLoading(model?.rowsLoadingState);
```

Comment:
The `?` is not necessary here.

Reply:
`modelLoading` has been removed as this check was resulting in selenium test failures for the grid panel remounting when we didn't want it to on reloading of rows.